utility/test/module_test.go

```diff
@@ -67,7 +66,19 @@ func NewTestingUtilityContext(t *testing.T, height int64) utility.UtilityContext
 	}
 }
 
-func newTestPersistenceModule(_ *testing.M, databaseUrl string) modules.PersistenceModule {
+func newTestingUtilityContextWithPersistenceContext(t *testing.T, height int64, persistenceContext modules.PersistenceRWContext) utility.UtilityContext {
+	return utility.UtilityContext{
+		LatestHeight: height,
+		Mempool:      NewTestingMempool(t),
+		Context: &utility.Context{
+			PersistenceRWContext: persistenceContext,
+			SavePointsM:          make(map[string]struct{}),
+			SavePoints:           make([][]byte, 0),
+		},
+	}
+}
+
+func newTestPersistenceModule(m *testing.M, databaseUrl string) modules.PersistenceModule {
```

Comment:
Why do we still have the existing persistence module here? Is it safe to assume that this will be removed by this issue?